diff --git a/node_modules/swiper/types/components/navigation.d.ts b/node_modules/swiper/types/components/navigation.d.ts
index 5b93fcc..58902d0 100644
--- a/node_modules/swiper/types/components/navigation.d.ts
+++ b/node_modules/swiper/types/components/navigation.d.ts
@@ -36,7 +36,7 @@ export interface NavigationOptions {
    *
    * @default null
    */
-  nextEl?: CSSSelector | HTMLElement;
+  nextEl?: CSSSelector | HTMLElement | null;
 
   /**
    * String with CSS selector or HTML element of the element that will work
@@ -44,7 +44,7 @@ export interface NavigationOptions {
    *
    * @default null
    */
-  prevEl?: CSSSelector | HTMLElement;
+  prevEl?: CSSSelector | HTMLElement | null;
 
   /**
    * Toggle navigation buttons visibility after click on Slider's container
